pkg/volume/volume.go

```diff
@@ -28,6 +28,28 @@ import (
 type Volume interface {
 	// GetPath returns the directory path the volume is mounted to.
 	GetPath() string
+
+	// VolumeMetricsProvider embeds methods for exposing Volume metrics (e.g. capacity).
+	VolumeMetricsProvider
+}
+
+// VolumeMetricsProvider exposes metrics (e.g. capacity) related to a Volume.
+type VolumeMetricsProvider interface {
```

Comment:
don't stutter - `volume.MetricsProvider` is better than `volume.VolumeMetricsProvider`